run_bot.sh

```
@@ -1,1082 +1,194 @@
-    local image_name=${DOCKER_IMAGE_NAME:-meet-teams-bot:latest}
-    echo "$image_name"
+    echo "${DOCKER_IMAGE_NAME:-meet-teams-bot:latest}"
 }
 
 # Find available port
-find_available_port() {
-    local start_port=${1:-3000}
-    local port=$start_port
+find_port() {
+    local port=${1:-3000}
     while lsof -Pi :$port -sTCP:LISTEN -t >/dev/null 2>&1; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)